polymode-flycheck.el

```diff
@@ -0,0 +1,162 @@
+;;; polymode-flycheck.el --- Flycheck helpers for polymodes -*- lexical-binding: t -*-
+;;
+;; Copyright (C) 2013-2018, Vitalie Spinu
+;; Author: Vitalie Spinu
```

Comment:
You need to add yourself as an author. 

BTW, do you have signed papers for FSF? I would like to contribute the core polymode to ELPA in the near future.

Reply:
No, I didn't sign papers.